README.md

```diff
@@ -12,6 +12,7 @@ To add a conference to the list, please [contribute](#contributing)!
 | [JSFoo Bangalore](https://hasgeek.com/jsfoo/2019/) | 27-28 Sep | NIMHANS Convention Centre, Bangalore | JSFoo is in its ninth edition this year. The 2019 edition is a two-track event with talks and Birds of Feather (BOF) sessions | No |
 | [React India](https://www.reactindia.io/) | 26-28 Sep | Goa | A 3 day conference with first day focussing on workshops & the next 2 days talks on things around React, React Native, GraphQL, VR & AR, Redux, Preact and more. | [Yes](https://www.notion.so/Scholarship-Program-React-India-2019-490794dd0a464db0a43b4c8c5440839f) |
 | [JSFoo: VueDay](https://hasgeek.com/jsfoo/2019-vueday/proposals) | 30 Aug | Bangalore International Centre | A day spent discussing Vue.js, developments in the Vue ecosystem and component architecture | Yes, for speakers |
+| [WordCamp Nagpur 2019](https://2019.nagpur.wordcamp.org) | 11 Aug | MIA Centre For Sports & Recreation- Nagpur | WordCamps are casual conferences that are local in nature. Topics cover everything related to WordPress and include things such as blogging, coding, branding, and security. | Yes, for speakers |
```

Comment:
- Can you remove the 2019 from the title? It's implicit :)

- nit: `MIA Center for Sports & Recreation, Nagpur` instead of `MIA Centre For Sports & Recreation- Nagpur`

- I think the `WordCamps are casual conferences that are local in nature.` can be removed from the description. Let's keep it small and concise as much as possible.